src/views/Stake/Stake.jsx

```diff
@@ -27,14 +27,14 @@ import CheckCircleIcon from "@material-ui/icons/CheckCircle";
 import RebaseTimer from "../../components/RebaseTimer/RebaseTimer";
 import TabPanel from "../../components/TabPanel";
 import { trim, getTokenImage, getPairImage, getOhmTokenImage } from "../../helpers";
-import { changeStake, changeApproval } from "../../actions/Stake.actions";
-import { getFraxData } from "../../actions/App.actions";
+import { changeStake, changeApproval } from "../../slices/StakeThunk";
```

Comment:
Why is this `StakeThunk` vs `MainSlice`?

Reply:
everything else seems to be slice.

Reply:
Its stakeThunk because I called the files AccountSlice and BondSlice if they actually use the createSlice method, defining a substate.  If instead they only use createAsyncThunk to define thunks without making a slice, then I called them stakeThunk and migrateThunk